service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillShowAction.kt

```diff
@@ -178,8 +178,18 @@ class BackfillShowAction @Inject constructor(
                       )
                     }
                     td("hidden py-5 pl-8 pr-0 text-right align-top tabular-nums text-gray-700 sm:table-cell") { +"""${partition.matching_records_per_minute} #/m""" }
-                    // TODO properly calculate the ETA until finished
-                    td("py-5 pl-8 pr-0 text-right align-top tabular-nums text-gray-700") { +"""ETA TODO""" }
+                    td("py-5 pl-8 pr-0 text-right align-top tabular-nums text-gray-700") {
+                      when {
+                        partition.state != BackfillState.RUNNING -> +"-"
+                        !partition.precomputing_done -> +"Computing size..."
```

Comment:
```suggestion
                        !partition.precomputing_done -> +"Computing..."
```

Less text is better given it is a narrow column